packages/ui/package.json

```
@@ -21,37 +21,43 @@
 		"prepublishOnly": "pnpm run package",
 		"prepare": "svelte-kit sync",
 		"storybook": "storybook dev --no-open -p 6006",
-		"storybook:build": "storybook build"
+		"storybook:build": "storybook build",
+		"test": "vitest run --mode development",
+		"test:watch": "vitest --watch --mode development"
 	},
 	"devDependencies": {
 		"@csstools/postcss-bundler": "^1.0.15",
-		"@storybook/addon-docs": "^8.2.7",
-		"@storybook/addon-essentials": "^8.2.7",
-		"@storybook/addon-links": "^8.2.7",
-		"@storybook/blocks": "^8.2.7",
-		"@storybook/svelte": "^8.2.7",
-		"@storybook/sveltekit": "^8.2.7",
+		"@storybook/addon-docs": "^8.3.0",
+		"@storybook/addon-essentials": "^8.3.0",
+		"@storybook/addon-links": "^8.3.0",
+		"@storybook/blocks": "^8.3.0",
+		"@storybook/experimental-addon-vitest": "^8.3.0-beta.1",
```

I think you forgot this one @Caleb-T-Owens 
```suggestion
		"@storybook/experimental-addon-vitest": "^8.3.0",
```

---

I tried, but it doesn't seem like there is a release version of it out yet @JReinhold.

https://www.npmjs.com/package/@storybook/experimental-addon-vitest?activeTab=versions

There seems to be this vite error that you can now see in CI that also seems to have started to occur with my changes which I need to look into

---

Perhaps we ought to be using https://www.npmjs.com/package/@storybook/experimental-addon-test instead of https://www.npmjs.com/package/@storybook/experimental-addon-vitest

---

Ah yes sorry! The name changed last minute, I forgot